public/antigravity.css

```
@@ -380,20 +382,122 @@ body.antigravity-theme .library-empty a:hover {
+  stroke: var(--ag-violet);
+  stroke-width: 1.4;
+  vector-effect: non-scaling-stroke;
+}
+
+html[data-theme="dark"] body.antigravity-theme .library-empty--ignition .witness-anchor__shape {
+  stroke: var(--ag-violet);
+}
+
+body.antigravity-theme .library-empty-headline {
+  margin: 0;
+  font-family: 'Outfit', 'Manrope', 'Inter', sans-serif;
```

Stylelint (17.11.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)